src/components/HeaderLogo.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import '../App.css';
+
+const Header = () => {
+    return (
+        <div className="Header">
```

Comment:
You could/should just make this a `<header>` tag instead of a `div`.

Reply:
true that! good catch on semantics. Must. Refrain. Using. Divs.